data-manipulation/encryption/hc-128/encrypt-data-using-hc-128-via-wolfssl.yml

```diff
@@ -0,0 +1,23 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using HC-128 via WolfSSL
+    namespace: data-manipulation/encryption/hc-128
+    author: blaine.stancill@mandiant.com
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::HC-128 [C0027.006]
+    references: https://github.com/wolfSSL/wolfssl/blob/master/wolfcrypt/src/hc128.c
```

Comment:
can you make this a link to a specific hash, so that the line number references below won't change?

Reply:
```suggestion
    references: https://github.com/wolfSSL/wolfssl/blob/6694775d4b8c22e7411ec69e8a2b516f107b97ef/wolfcrypt/src/hc128.c
```